CONTRIBUTING.rst

```diff
@@ -0,0 +1,109 @@
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given. 
+
+You can contribute in many ways:
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/learningequality/kolibri/issues.
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the GitHub issues for bugs. Anything tagged with "bug"
+is open to whoever wants to implement it.
+
+Implement Features
+~~~~~~~~~~~~~~~~~~
+
+Look through the GitHub issues for features. Anything tagged with "feature"
+is open to whoever wants to implement it.
+
+Write Documentation
+~~~~~~~~~~~~~~~~~~~
+
+Kolibri could always use more documentation, whether as part of the 
+official Kolibri docs, in docstrings, or even on the web in blog posts,
+articles, and such.
+
+Submit Feedback
+~~~~~~~~~~~~~~~
+
+The best way to send feedback is to file an issue at https://github.com/learningequality/kolibri/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Here's how to set up `kolibri` for
+local development.
+
+1. Fork_ the `kolibri` repo on GitHub.
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/kolibri.git
+
+3. Create a branch for local development::
+
+    $ git checkout -b name-of-your-bugfix-or-feature
+
+Now you can make your changes locally.
+
+4. When you're done making changes, check that your changes pass style and unit
+   tests, including testing other Python versions with tox::
+
+    $ tox
+
+To get tox, just pip install it.
+
+5. Commit your changes and push your branch to GitHub::
+
+    $ git add .
+    $ git commit -m "Your detailed description of your changes."
+    $ git push origin name-of-your-bugfix-or-feature
+
+6. Submit a pull request through the GitHub website.
+
+.. _Fork: https://github.com/learningequality/kolibri/fork
+
+Pull Request Guidelines
+-----------------------
+
+Before you submit a pull request, check that it meets these guidelines:
+
+1. The pull request should include tests.
+2. If the pull request adds functionality, the docs should be updated. Put
+   your new functionality into a function with a docstring, and add the
+   feature to the list in README.rst.
+3. The pull request should work for Python 2.6, 2.7, and 3.3, and for PyPy.
+   Check https://travis-ci.org/learningequality/kolibri 
+   under pull requests for active pull requests or run the ``tox`` command and
+   make sure that the tests pass for all supported Python versions.
```

Comment:
True, in this particular case, the whole text is a standard text derived from the cookiecutter template. I think it's a nice inspiration and the **Contributing** chapter title could stay -- with a Developer Guide inside the Contributing chapter (not on the same page, but as a separate item) ? Something like:

 * Contributing
    * Reporting issues
    * Developer Guide
    * Testing
    * Styleguide